README.md

```diff
@@ -1,6 +1,8 @@
 oVirt Virtual Machine Infrastructure
 ====================================
 
+## THIS ROLE IS DEPRECATED (please go to https://github.com/ovirt/ovirt-ansible-collection)
```

Comment:
I'd use something like:

`This role has been migrated to oVirt Ansible Collection, please use latest version from there. This repository is now readonly and no longer used for active development`